pkg/apis/eventing/v1beta1/broker_defaults.go

```diff
@@ -18,13 +18,29 @@ package v1beta1
 
 import (
 	"context"
+	"fmt"
+
+	"knative.dev/eventing/pkg/apis/config"
+	"knative.dev/pkg/apis"
 )
 
 func (b *Broker) SetDefaults(ctx context.Context) {
 	// TODO(vaikas): Set the default class annotation if not specified
-	b.Spec.SetDefaults(ctx)
+	fmt.Printf("GOT CONTEXT AS: %+v %#v", ctx, ctx)
+	withNS := apis.WithinParent(ctx, b.ObjectMeta)
+	b.Spec.SetDefaults(withNS)
 }
 
 func (bs *BrokerSpec) SetDefaults(ctx context.Context) {
-	// None
+	if bs.Config != nil {
+		return
+	}
+
+	cfg := config.FromContextOrDefaults(ctx)
+	fmt.Printf("GOT CONFIG AS: %+v", cfg)
+	fmt.Printf("GOT CONTEXT AS: %+v %#v", ctx, ctx)
+	c, err := cfg.Defaults.GetBrokerConfig(apis.ParentMeta(ctx).Namespace)
+	if err != nil {
```

Comment:
```suggestion
	if err == nil {
```

Reply:
LOL :) Thanks :) 👍 I wish I had seen that myself LOL :)